README.md

```diff
@@ -1,99 +1,63 @@
-# backstage-operator
-Operator for deploying Backstage for Janus-IDP.
+# Backstage Operator
 
-## Description
-Implementing https://janus-idp.io/docs/deployment/k8s/ procedure
-At first stage CR update does not affect Backstage Objects, just installation (same as Helm)
-TODO: Do we need to continuosly sync the states? Which way if so: from CR to Objects or back or (somehow) back and forth?
-
-Make sure namespace is created.
-
-Local Database (PostgreSQL) is created by default, to disable
-spec: 
-  skipLocalDb: true
-This way third party DB can theorethically be configured. TODO: It just requires some changes in Backstage appConfig (I think), 
-because it only expects either in-container SQLite or MySQL.
-TODO: should we consider using in-container SQLite for K8s deployment as well (single container deployment)?
-
-TODO: POSTGRES_HOST = <name-of the service> , POSTGRES_PORT = <port>[5432] can be delivered to the Backstage 
-Deployment out of Postgres Secret? Indeed, it is not really a secret.
+## The Goal
+The Goal of [Backstage](https://backstage.io) Operator project is creating Kubernetes Operator for configuring, installing and synchronizing Backstage instance on Kubernetes/OpenShift. 
```

Comment:
I don't think we're targetting k8s, only openshift. IIRC the helmchart is for k8s (and openshift too) but the operator is ONLY for openshift. @jasperchui @christophe-f  am I correct here?

Reply:
@nickboldt the codebase is working and tested on both K8s AND Openshift.
This is the message for GitHub repository README.
We probably need to slightly change this message and we definitely need to add more information about openshift/operatorhub as soon as we have it, but technically it is the truth and I doubt it will be changed by the same reason as helm chart I guess (support of openshift == support of k8s + Route for this operator)

So, I'd definitely welcome improving the message from the marketing standpoint if it is the goal here but technically the current message it is relevant :)

Reply:
I'll let @christophe-f weigh in from the PM side. I'm OK with *technically correct* docs, but we don't want to imply support that we won't actually do.